pkgs/tools/misc/oh-my-posh/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, fetchFromGitHub
+, fetchurl
+, buildGoModule
+, unzip
+, getThemes ? false
+}:
+
+let
+  owner = "JanDeDobbeleer";
+  pname = "oh-my-posh";
+  version = "7.72.5";
+  themeIntallSteps = let themes = fetchurl {
+    url = "https://github.com/${owner}/${pname}/releases/download/v${version}/themes.zip";
+    sha256 = "F0CeospZmZEoM1re5p/A/aQECenRIjIHMr66WuPEeNg=";
+  }; in
+    ''
+      mkdir -p $out/themes
+      ${unzip}/bin/unzip ${themes} -d $out/themes
+    '';
+
+in
+
+buildGoModule rec {
+  inherit version;
+  inherit pname;
+  inherit owner;
+  src = fetchFromGitHub
+    {
+      inherit owner;
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "D1dR/S/qyTHyCZt7IvIhmYsq2Yh89Wm0PoGvSsko0jA=";
+    } + "/src";
+
+  vendorSha256 = "AzfNq6t+Zk4eI0v6lWTzpcEmHPxF/3Lrk9g/OgOoTmI=";
+
+  ldflags = [ "-a" "-s" "-w" "-X" "main.Version=${version}" "-extldflags" ''"-static"'' ];
+
+  tags = [ "netgo" "osusergo" "static_build" ];
+
+  dontFixup = true;
```

Comment:
Why? (add a comment)

Reply:
@balsoft I took tags directly from [release scripts](https://github.com/JanDeDobbeleer/oh-my-posh/blob/v7.72.5/src/.goreleaser.yml); and took a guess that [definition of buildGoModule](https://github.com/NixOS/nixpkgs/blob/7beebb590d541ffa534aa34b0b163f81c3c72c2c/pkgs/development/go-modules/generic/default.nix#L13) would have a corresponding attribute.

And fixup tries to `patchELF` a statically-linked binary,  which does not have much sense, does it? I do not have a strong opinion on this. In the end of the day, the executable still works.

Reply:
Please remove it. This also disabled other hooks which is most likely unwanted.